.github/workflows/lint.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint
+
+on: 
+  push:
+    branches:
+      - main
+
+jobs:
+  lint:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 16
+
+    - run: yarn install
+
+    - name: Lint
+      run: yarn lint --fix
```

Comment:
```suggestion
      run: yarn lint --fix

```

endline

Reply:
I will work on the changes. I have encountered a Restyled error for my commit. I am unable to understand where I went wrong. Can you suggest me the changes so that I can fix it

Reply:
ignore the restyled error, it is irrelevant